pkg/runtime/runtime.go

```diff
@@ -171,7 +186,44 @@ func (p provider) CreateSandbox(podName string,
 		}
 	}
 
+	// Must create a Network policy to allow for comunication from the APB pod to the target namespace.
+	networkPolicy := &networkingv1.NetworkPolicy{
```

Comment:
is there a "deny all ingress" network policy created in the apb pod namespace that will prevent the user from initiating communication in the other direction?

Reply:
Correct me if I am wrong, but if the network policy does not exist to allow the traffic the base state of the plugin `ovs-networkpolicy` is to disallow traffic.

If you are using `ovs-multitenant` plugin I don't think that you can.

@danwinship are these statements correct?

Reply:
The base state of the networkpolicy plugin is to allow all traffic. But you said before there's nothing listening to any ports in the temp namespace, so this shouldn't matter?

But if you want to be paranoid, you could add a policy:

    kind: NetworkPolicy
    apiVersion: extensions/v1beta1
    metadata:
      name: deny-all
    spec:
      podSelector:
    ingress: []

Reply:
I think that if we ever open a port for the APB  to listen on then we would want to control the network with `NetworkPolicy` as @danwinship said. I don't think it is a concern as there is nothing exposed to the network in the apb pod namespace. 

@liggitt and I think @enj you had the same concern, does the above sound acceptable to you?

Reply:
yes